### PR TITLE
Move CTT calculation from BusPwm to Bus class.

### PR DESCRIPTION
**Pull request by Robert-github-com**
This enable it to other bus types.
Need for adding RGBWW LEDs on digital bus.

### CODE REVIEW

File: wled00/bus_manager.h [hunk 1]
```diff
@@ -150,6 +155,32 @@ class Bus {
         if (_cctBlend > WLED_MAX_CCT_BLEND) _cctBlend = WLED_MAX_CCT_BLEND;
       #endif
     }
+    static void calculateCCT(uint32_t c, uint8_t &ww, uint8_t &cw) {
```
Comment (blazoncek) [inline wled00/bus_manager.h:158]: This can be a private or protected, does not need to be public method.

File: wled00/bus_manager.h [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 #include "const.h"
 
+//colors.cpp
+uint32_t colorBalanceFromKelvin(uint16_t kelvin, uint32_t rgb);
```
Comment (blazoncek) [inline wled00/bus_manager.h:11]: Why did you move all three into .h? There is no need to do that, only move a needed one.

**Review by blazoncek — COMMENTED** [review]
Apart from minor suggestions all seems fine.

File: wled00/bus_manager.h (hunk 1 above)
Comment (Robert-github-com) [reply wled00/bus_manager.h:176]: Done

File: wled00/bus_manager.h (hunk 2 above)
Comment (Robert-github-com) [reply wled00/bus_manager.h]: Moved back

**Comment by deece** [comment]
@blazoncek With the merge of #3810, this can now be closed
